### ISSUE TITLE
Добавить username в database.yml

### ISSUE THREAD

**Issue by Lindemann66** [fortweety/rails_gb#2]
Не хватает обозначения `username`, например `username: fortweety`
При этом, для окружения production `username` почему-то есть)

**Comment by fortweety** [comment]
ага, а можно будет еще по pg потом обратиться? а то никак не могу с ним справиться

**Comment by Lindemann66** [comment]
Конечно, предлагаю обсудить в слаке. Ссылка на вступление (если вы еще не присоединились) в обсуждении первого урока
